.circleci/config.yml

```diff
@@ -29,10 +29,20 @@ jobs:
 
 workflows:
   version: 2
-
-  build_and_publish:
+  build-only:
     jobs:
-      - build
+      - build:
+          filters:
+            tags:
+              ignore: /^v.*/
+  build-and-publish:
+    jobs:
+      - build:
+          filters:
+            tags:
+              only: /^v.*/
+            branches:
```

Comment:
Does it build all branches by default? I'm asking because I'm wondering if the `tags. only: /^v.*/` would be enough, or you need to add the `ignore` otherwise it will build all branche by default 🤷‍♂️

Reply:
it does build all branches but as part of the conda build it imports the built library and runs the tests, it just doesn't upload the build anywhere unless this is a release tagged `v.xxx` 

This is good because it gives us tests CI for "free"

Reply:
Thanks for the chat. I now understand better the default CircleCI behaviour.

As said, feel free to merge in this current state 👍